.github/copilot-instructions.md

```
@@ -0,0 +1,56 @@
+# hass-opnsense Home Assistant Integration Copilot Instructions
+
+## General Guidelines
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)